docs/docs/documents/predefined-pages.mdx

```diff
@@ -0,0 +1,270 @@
+
+@Module({})
+export class AppModule {
+    static forRoot(config: Config): DynamicModule {
+        return {
+            module: AppModule,
+            imports: [
+                /* ... */,
+                PageTreeModule.forRoot({
+                    PageTreeNode: PageTreeNode,
+                    Documents: [
+                        Page,
+                        Link,
++                       PredefinedPage
+                    ],
+                    Scope: PageTreeNodeScope,
+                }),
+            ],
+        };
+    }
+}
+```
+
+### Admin
+
+The following steps are required in the Admin:
+
+1. Creation of the predefined pages module
+2. Registration of the `PredefinedPage` document type
+
+#### Creation of the predefined pages module
+
+Create the module in `src/documents/predefinedPages`.
+To do this, copy the corresponding [folder](https://github.com/vivid-planet/comet/tree/main/demo/admin/src/documents/predefinedPages) from the Demo project.
+The following files should be copied:
+
+-   `EditPredefinedPage.tsx`: The editing component
+-   `EditPredefinedPage.gql.ts`: The GraphQL operations for the edit component
+-   `PredefinedPage.tsx`: The document type
+-   `PredefinedPageLabels`: The labels for the predefined page types
+
+#### Registration of the `PredefinedPage` document type
+
+Add the newly created document type to the page tree document types:
+
+```diff title"=src/common/MasterMenu.tsx"
+import { Link } from "@src/documents/links/Link";
+import { Page } from "@src/documents/pages/Page";
++ import { PredefinedPage } from "@src/documents/predefinedPages/PredefinedPage";
+
+export const pageTreeDocumentTypes: Record<string, DocumentInterface<any, any>> = {
+    Page,
+    Link,
++   PredefinedPage
+};
+```
+
+### Site
+
+The following steps are required in the Site:
+
+1. Creation of the predefined pages module
+2. Adding the predefined pages to the middleware
+3. Adding the predefined page document type to the `PageLink`
+
+#### Creation of the predefined pages module
+
+Create the module in `src/documents/predefinedPages`.
+To do this, copy the corresponding [folder](https://github.com/vivid-planet/comet/tree/main/demo/site/src/documents/predefinedPages) from the Demo project.
+The following files should be copied:
+
+-   `predefinedPagePaths.ts`: The paths where the predefined pages are located in the code
+
+#### Adding the predefined pages to the middleware
+
+We use a combination of a redirect and a rewrite in the middleware to ensure that a predefined page can be viewed at the expected location in the page tree.
+
+**Example:** The news page in the code at `https://example.com/news` should be rendered in the page tree at `https://example.com/company/news`.
+We need the following redirect and rewrite:
+
+-   A **rewrite** from `/company/news` to `/news` (to ensure that the page can be viewed at the expected location)
+-   A **redirect** from `/news` to `/company/news` (to avoid duplicate content)
```

Comment:
I would rather use translation of the url as example, as adding a fixed /company in the url should be done using static routes instead

Reply:
ecd2e4880e7d2c193d62f16ef1ce4585d637d867